### DISCUSSION TITLE
C5336AD1-F013-7FF3-441D-0EF12E0BCB22

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#42] [General]
# Gliese 892

Gliese 892 (also known as HD 219134) is a K-type main-sequence star 21.336 light-years away in the constellation Cassiopeia. A total of six planets (2 rocky super-Earths, 3 Neptunians, and 1 Jovian) are found orbiting this star.

https://celestia.mobi/addon?item=C5336AD1-F013-7FF3-441D-0EF12E0BCB22&lang=en

**Comment by exl001** [comment]
Very nice!

**Comment by exocritic5457** [comment]
Not much I can say, but I easily approve.

**Reply by exocritic5457** [reply]
However, didn't g get disproven? neither of its periods matched.
